documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -293,6 +293,17 @@ Supported values are:
 If a component model is given for a specific mapper via `@Mapper#componentModel()`, the value from the annotation takes precedence.
 |`default`
 
+|`mapstruct.defaultInjectionStrategy`
+| The type of the injection in mapper via parameter `uses`.
+
+Supported values are:
+
+* `field`: dependencies will be injected in fields
+* `constructor`: dependencies will be injected via constructor.
+
+If a injection strategy is given for a specific mapper via `@Mapper#injectionStrategy()`, the value from the annotation takes precedence.
```

Comment:
what do you mean here?

Reply:
I wrote it like with component model from our documentation:
"If a component model is given for a specific mapper via @Mapper#componentModel(), the value from the annotation takes precedence." . Was I wrong?

Reply:
No you are not wrong.. But perhaps add over what it takes precedence. So: `...takes precedence over the option`. Although it should probably be clear from the context as well.